### PR TITLE
Inline documentation for test.py file added

### PR DESCRIPTION
**Pull request by Kratos-is-here**
** Inline documentation for test.py added

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -1,153 +1,179 @@
...
+from fedrec.utilities import random_state, registry
+from fedrec.utilities.serialization import dash_separated_floats
+
+
+class InferenceMaker:
+    """
+    Tests our model on the testing dataset (test-loader)
+    """
+    def __init__(self, config, devices, log_dir) -> None:
+        """The __init__ method lets the class initialize the objects attributes
```
Comment (ramesht007) [inline test.py:17]: Add ```Args``` above the init method.
and you can remove the comment for init method.

**Review by ramesht007 — COMMENTED** [review]
Good work !. Some minor changes then we are good to go.
Also fix the characters in a single line, according to PEP8 guide guidelines we can only have upto 79 character in a line.

**Comment by Kratos-is-here** [comment]
Made the required changes in test.py.

File: test.py [hunk 2]
```diff
@@ -1,153 +1,183 @@
...
+from argparse import ArgumentParser
+
+import numpy as np
+import torch
+import yaml
+from sklearn import metrics
+
+from fedrec.utilities import random_state, registry
+from fedrec.utilities.serialization import dash_separated_floats
+
```
Comment (vkkhare) [inline test.py:10]: Why are there changes in the entire file?

File: test.py [hunk 3]
```diff
@@ -1,153 +1,183 @@
...
+
+class InferenceMaker:
+    """
+    Tests our model on the testing dataset (test-loader)
+    
+    Args:
```
Comment (vkkhare) [inline test.py:16]: Follow [Numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html)

Arguments not args

**Review by vkkhare — CHANGES_REQUESTED** [review]

**Comment by Kratos-is-here** [comment]
@vkkhare I have updated test.py

**Pull request closed without merging**